bin/regenerate-offline

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -eu
+
+GOVUK_FRONTEND_PREFIX=https://github.com/alphagov/govuk-frontend/releases/download/v${RELEASE}/
+GOVUK_FRONTEND_ARCHIVE_NAME=release-v${RELEASE}.zip
+GOVUK_FRONTEND_URL="${GOVUK_FRONTEND_PREFIX}${GOVUK_FRONTEND_ARCHIVE_NAME}"
+
+wget "${GOVUK_FRONTEND_URL}"
+if [[ ! -f "${GOVUK_FRONTEND_ARCHIVE_NAME}" ]]; then
+  echo "${GOVUK_FRONTEND_ARCHIVE_NAME} does not exist."
+  exit 1
+else
+  unzip "${GOVUK_FRONTEND_ARCHIVE_NAME}" -d ./offline
+fi
+mv ./offline/govuk-frontend-"${RELEASE}".min.css ./offline/govuk-frontend.min.css
+mv ./offline/govuk-frontend-ie8-"${RELEASE}".min.css ./offline/govuk-frontend-ie8.min.css
```

Comment:
i dont think this is needed as page is so simple and we dont support ie8

Reply:
Thanks. I've removed IE8 and JS files